flexget/plugins/modify/path_select.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import os
+import random
+from collections import namedtuple
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+
+log = logging.getLogger('path_select')
+
+SYMBOLS = {
+    'customary': ('B', 'K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'),
+    'customary_ext': ('byte', 'kilo', 'mega', 'giga', 'tera', 'peta', 'exa', 'zetta', 'iotta'),
+    'iec': ('Bi', 'Ki', 'Mi', 'Gi', 'Ti', 'Pi', 'Ei', 'Zi', 'Yi'),
+    'iec_ext': ('byte', 'kibi', 'mebi', 'gibi', 'tebi', 'pebi', 'exbi', 'zebi', 'yobi'),
```

Comment:
I don't think we need all these possibilities, I think we should just define a standard and document it. The one thing I was also wondering about is what to do about decimal vs binary units... mostly, whether MB should actually mean MiB or not...